src/server.c

```diff
@@ -2766,6 +2766,16 @@ void redisOpArrayFree(redisOpArray *oa) {
 
 /* ====================== Commands lookup and execution ===================== */
 
+int isContainerCommandBySdsLogic(dict *commands, sds s) {
+    struct redisCommand *base_cmd = dictFetchValue(commands, s);
+    int has_subcommands = base_cmd && base_cmd->subcommands_dict;
+    return has_subcommands;
+}
+
+int isContainerCommandBySds(sds s) {
+    return isContainerCommandBySdsLogic(server.commands, s);
+}
```

Comment:
i think we don't need the wrapper function, and maybe not the other one.
it's a simple 2 line check that can be done directly on the caller's code (specifically since it's only used in one place)

Reply:
I removed the other one